File: examples/python/automations/outlook/get.py
from dotenv import load_dotenv
from retab import Retab

assert load_dotenv("../../../.env.production")

reclient = Retab()

automation = reclient.processors.automations.outlook.get(
    outlook_id="outlook_DDw72RgWgIfKFXGaWXcGu",
)

print(automation.model_dump_json(indent=2))

File: cookbook/typescript/automations/outlook/get.ts
import { config } from 'dotenv';
import { Retab } from '@retab/node';



async function main() {
  config({ path: '../../../.env.production' });
  
  const reclient = new Retab();
  
  const automation = await reclient.processors.automations.outlook.get({
    outlook_id: "outlook_DDw72RgWgIfKFXGaWXcGu",
  });
  
  console.log(JSON.stringify(automation, null, 2));
  
}

main().catch(console.error);